libs/features/components/ui-lists/src/lib/elewa-group-article-list/elewa-group-article-list.component.ts

```diff
@@ -1,5 +1,5 @@
 import { Component, Input } from '@angular/core';
-export interface Article { image: string; timestamp: string; title: string; body: string;} //import this interface in the component you will be reusing article-list in. e.g import { Article } from 'libs/features/components/ui-lists/src/lib/elewa-group-article-list/elewa-group-article-list.component';
+export interface Article { image: string; timestamp: string; title: string; body: string; topic: string;} //import this interface in the component you will be reusing article-list in. e.g import { Article } from 'libs/features/components/ui-lists/src/lib/elewa-group-article-list/elewa-group-article-list.component';
```

Comment:
Move interface to an interface file inside this lib. models/article.interface.ts